wandb/integration/keras/callbacks/tables_builder.py

```diff
@@ -124,7 +136,10 @@ def on_epoch_end(self, epoch: int, logs: Optional[Dict[str, float]] = None) -> N
         # Log the pred_table as W&B Artifacts
         self.log_pred_table()
```

Comment:
Not checking if `in_memory = True` is set here ? Maybe I'm understanding it wrong. But When `in_memory =True` is set shouldn't it only log at the end of training ?

Reply:
@parambharat The check is actually being done inside [`self.log_pred_table`](https://github.com/wandb/wandb/tree/feat/wandbevalcallback).